lib/util/requestAssertions.js

```diff
@@ -120,10 +121,54 @@ module.exports = {
    */
   assertHasIndexAndCollection: request => {
     if (!request.input.resource.index) {
-      throw new BadRequestError(`${request.input.controller}:${request.input.action} must specify an index.`);
+      throw new BadRequestError(`${getRequestRoute(request)} must specify an index.`);
     }
     if (!request.input.resource.collection) {
-      throw new BadRequestError(`${request.input.controller}:${request.input.action} must specify a collection.`);
+      throw new BadRequestError(`${getRequestRoute(request)} must specify a collection.`);
+    }
+  },
+
+  /**
+   * @param {Request} request
+   */
+  assertHasStrategy: function requestAssertHasStrategy (request) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Use shorthand for method "assertHasStrategy". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3498)